### PR TITLE
Implement view movie in isolation and respective tests

### PR DESCRIPTION
**Pull request by TemboJembo**
Added tests and created the html page for task 4.

### CODE REVIEW

File: templates/get_single_movie.html [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
-<h1 class="mb-5"></h1>
-<!-- TODO: Feature 4 -->
+    <h1 class="mb-5">{{movie.title}}</h1>
+    <!-- TODO: Feature 4 -->
+    <h3>Director</h3>
+    <p>{{movie.director}}</p>
+    <h3>Rating</h3>
+    <p>{{movie.rating}}</p>
+    <br>
+    <form action='/movies/{{movie.movie_id}}/edit' method="get">
```
Comment (RileyRetzloff) [inline templates/get_single_movie.html:13]: I'm not sure but do the buttons need a 'name' attribute so the values from them can be read in? Also might be good to add some Bootstrap button styling on both buttons. 
image

File: tests/e2e/test_view_movie_page.py
Comment (RileyRetzloff) [inline tests/e2e/test_view_movie_page.py:1]: LGTM

File: tests/unit/test_get_movie_by_id.py
Comment (RileyRetzloff) [inline tests/unit/test_get_movie_by_id.py:1]: LGTM

File: app.py
Comment (RileyRetzloff) [inline app.py:1]: LGTM

**Review by RileyRetzloff — COMMENTED** [review]
Everything looks good to me except, and tbh I'm not 100% sure about this, I think you might want to add a 'name' attribute to the two buttons in your get_single_movie.html file so the values from the two buttons are usable. You also might wanna add some Bootstrap styling which should be really easy. The code in app.py and your e2e and unit tests looks great to me.

**Review by AveryNewkirk — CHANGES_REQUESTED** [review]
Make sure to fetch and merge all changes so you can link your feature to the project. Everything looks great but we cant merge this until there is a link.

**Review by AveryNewkirk — APPROVED** [review]
Looks great.

**Pull request merged**